regulation-worker/cmd/main_test.go

```diff
@@ -16,52 +15,53 @@ import (
 	"github.com/rudderlabs/rudder-server/config"
 	backendconfig "github.com/rudderlabs/rudder-server/config/backend-config"
 	main "github.com/rudderlabs/rudder-server/regulation-worker/cmd"
+	"github.com/rudderlabs/rudder-server/regulation-worker/internal/client"
 	"github.com/rudderlabs/rudder-server/regulation-worker/internal/model"
 	"github.com/rudderlabs/rudder-server/utils/logger"
 	"github.com/stretchr/testify/require"
 )
 
 var (
-	c        = make(chan os.Signal, 1)
 	testData []test
 	mu       sync.Mutex
 )
 
 func TestMain(m *testing.M) {
 
-	os.Exit(run(m))
+	// os.Exit(run(m))
 }
 
 func handler() http.Handler {
 	srvMux := mux.NewRouter()
-	srvMux.HandleFunc("/worker/workspaces/{workspace_id}/regulations/worker-job", getJob).Methods("GET")
-	srvMux.HandleFunc("/worker/workspaces/{workspace_id}/regulations/worker-job/{job_id}", updateJobStatus).Methods("PATCH")
+	srvMux.HandleFunc("/dataplane/workspaces/{workspace_id}/regulations/workerJobs", getJob).Methods("GET")
+	srvMux.HandleFunc("/dataplane/workspaces/{workspace_id}/regulations/workerJobs/{job_id}", updateJobStatus).Methods("PATCH")
 
 	return srvMux
 }
 
 func run(m *testing.M) int {
```

Comment:
Seems like this method is unused now, maybe this can be removed.

Reply:
will be using this later